pkgs/tools/networking/openssh/common.nix

```diff
@@ -18,11 +18,13 @@
 , zlib
 , openssl
 , libedit
+, ldns
 , pkg-config
 , pam
 , libredirect
 , etcDir ? null
 , withKerberos ? true
+, withLdns ? true
 , libkrb5
```

Comment:
```suggestion
, libkrb5
, withLdns ? true
, ldns
```
FYI the sorting is intentionally that the packages are near the option

Reply:
well but then it really isn't consistent, because it's not that way for FIDO and PAM, but I can change the sorting for ldns and kerberos

Reply:
Or should I fix it for pam and fido2 as well when I'm already at it?